OrbitService/ProcessList.cpp

```diff
@@ -18,14 +17,7 @@
 namespace orbit_service {
 
 ErrorMessageOr<void> ProcessList::Refresh() {
-  auto cpu_result = utils::GetCpuUtilization();
-  if (!cpu_result) {
-    return outcome::failure(absl::StrFormat("Unable to retrieve cpu usage of processes: %s",
-                                            cpu_result.error().message()));
-  }
-  std::unordered_map<int32_t, double> cpu_usage_map = std::move(cpu_result.value());
-
-  std::vector<Process> updated_processes;
+  decltype(processes_) updated_processes{};
```

Comment:
use explicit type here decltype is not very readable.

Reply:
Done.